lib/combineNodes.js

```diff
@@ -0,0 +1,367 @@
+'use strict';
+var Cesium = require('cesium');
+var jp = require('jsonpath');
+var _ = require('underscore');
```

Comment:
Underscore is an old, traditionally browser-based library meant to make up for missing features of the browser API and incompatibility from platform to platform.  For Node.js, there is only a single platform and 99% of functionality is either directly supported by the native API or is otherwise only a couple of lines of code.  Here's an article that tries to lay out some examples:  https://www.reindex.io/blog/you-might-not-need-underscore/ That article does use some ES2015 examples, which technically we can't use yet, but it's really just shorthand for stuff that can be done in ES5 already.  We should never use a library if there is an equivalent native API already available, it just makes the code slower and less uniform.

As for equals, I don't see it actually being called anywhere in this PR, but if we really need a deep equality function, something like https://www.npmjs.com/package/deep-equal is probably a better choice.  Also, if we are only using deep-equal in tests, then there's already jasimine.toEqual and assert.equals

Normally, we avoid generic deep comparisons in our code and prefer implementing custom equals functions directly (such as `Matrix4.equals` for example) because they are way faster.  If we need to compare completely unknown blobs, then of course we may need to have generic comparisons, but since gltf is a known schema, I'm not sure of why that would be the case.